src/ethoscope/core/roi.py

```diff
@@ -20,7 +20,8 @@ def __init__(self, polygon, idx, value=None, orientation = None, regions=None):
         :type idx: int
         :param value: an optional value to be save for this ROI (e.g. to define left and right side)
         :param orientation: Optional orientation Not implemented yet
-        :param regions: Optional sub-regions within the ROI. Not implemented yet
+        :param regions: Optional sub-regions within the ROI.
```

Comment:
We need to make it clear, in the doc, that `regions` is a 2d array and that regions is a greyscale map / LUT. I.e. each pixel is the value of region in X,Y